linkerd/docs/routers.md

```diff
@@ -186,9 +178,10 @@ Key | Default Value | Description
 hostConnectionPool | An empty object | see [hostConnectionPool](#host-connection-pool).
 tls | no tls | The router will make requests using TLS if this parameter is provided.  It must be a [client TLS](#client-tls) object.
 loadBalancer | [p2c](#power-of-two-choices-least-loaded) | A [load balancer](#load-balancer) object.
-retries | see [retries](#retries) | A [retry policy](#retries) for all clients created by this router.
+failFast | `false` | If `true`, connection failures are punished more aggressively. Should not be used with small destination pools.
+requeueBudget | see [retries](#retries) | A [requeue budget](#retries) for connection-level retries.
```

Comment:
It looks like `requestBudget` is of type `RetryBudgetConfig` rather than type `RetriesConfig`, so I think this should be:

```
budget | See [retry budget](#retry-budget-parameters) | A [requeue budget](#retry-budget-parameters) for connection-level retries.
```